stablehlo/reference/Ops.cpp

```diff
@@ -64,6 +64,21 @@ Tensor evalCeilOp(const Tensor &operand, Type resultType) {
   return result;
 }
 
+Tensor evalClampOp(const Tensor &min, const Tensor &operand, const Tensor &max, 
+                   Type resultType) {
+  Tensor result(resultType);
+  for (auto it = result.index_begin(); it != result.index_end(); ++it) {
+    Element min_element =
```

Comment:
Use camel case for variable names

Reply:
There are too many of them: create https://github.com/openxla/stablehlo/issues/1030 to track and fix all later.

Reply:
This is a fairly small change, so it probably won't be hard to fix this for this specific PR? In #1030, we can then do a comprehensive sweep and fix the rest.